openfisca_core/tools/test_runner.py

```diff
@@ -31,7 +31,22 @@ def import_yaml():
     return yaml, Loader
 
 
-TEST_KEYWORDS = {'absolute_error_margin', 'description', 'extensions', 'ignore_variables', 'input', 'keywords', 'max_spiral_loops', 'name', 'only_variables', 'output', 'period', 'reforms', 'relative_error_margin'}
+TEST_KEYWORDS = {
+    'absolute_error_margin',
+    'description',
+    'extensions',
+    'ignore_variables',
+    'input',
+    'keywords',
+    'max_spiral_loops',
+    'name',
+    'neutralized_variables',
+    'only_variables',
+    'output',
+    'period',
+    'reforms',
+    'relative_error_margin'
```

Comment:
@MattiSG : since there are already keys such as ignore_variables, only_variables, I would prefer to go to `neutralize_variables`.